src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -272,6 +272,8 @@ public function process_directives( string $html ): string {
 	 * it returns null if the HTML contains unbalanced tags.
 	 *
 	 * @since 6.5.0
+	 * @since 6.6.0 The function now adds a warning when the HTML contains unbalanced
+	 *              tags or SVG/Math with directives.
```

Comment:
As we finally removed the debug message about using directives inside `<svg>` or `<math>` tags, we would need to update this description.

```suggestion
	 * @since 6.6.0 The function displays a warning message when the HTML contains
	 *              unbalanced tags.
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6413/commits/a9f15d878cf9eb650dc4e7291e9e927efafa2760